packages/cerebral/typings/index.d.ts

```diff
@@ -0,0 +1,109 @@
+// <reference types="node" />
+// <reference types="function-tree" />
+
+
+declare module 'cerebral/devtools' {
```

Comment:
Are you sure about this? Redux is an easier case - they just have function and no sub-modules... Maybe I'm wrong, but I can't find an example project that has a structure more similar to Cerebral's